### PR TITLE
Adding NavbarComponent/js & css

### PR DESCRIPTION
**Pull request by mugunga007**

### CODE REVIEW

File: client/src/NavbarComponent/NavbarComponent.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+       </Box>
+
+        </div>
+         </Box>
+         </Grid>
+      </div>
+  )
+}
+
+export default Navbar
```
Comment (jassalss) [inline client/src/NavbarComponent/NavbarComponent.js:63]: Nice Work Shema. Why did not you use app-bar component from material ui (all css and layout is done for us by material ui)

**Review by jassalss — COMMENTED** [review]
Nice Work Shema. Why did not you use app-bar component from material ui (all css and layout is done for us by material ui)

File: client/src/NavbarComponent/NavbarComponent.js (hunk 1 above)
Comment (mugunga007) [reply client/src/NavbarComponent/NavbarComponent.js:63]: I did styling (css) using MUI default one, though some stylings couldn't be done with MUI so I had to make our own css. any better way can still be suggested though. Thank you.

Comment (mugunga007) [reply client/src/NavbarComponent/NavbarComponent.js:63]: Yes I got it, I will add app bar. Thank you.

File: client/src/NavbarComponent/NavbarComponent.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import '../NavbarComponent/NavbarComponent.css';
```
Comment (sina-jamshidi) [inline client/src/NavbarComponent/NavbarComponent.js:3]: Can we move these into makeStyles instead of using stylesheets

**Pull request closed without merging**